src/platform-includes/getting-started-sourcemaps/javascript.mdx

```diff
@@ -0,0 +1,7 @@
+## Get Readable Stack Traces in your Sentry Errors
```

Comment:
Can we say this more succinctly? This sounds very much like we're marketing to the user which we want to avoid in the docs. I've made a suggestion, but I'm not sure if that changes the meaning at all. 
```suggestion
## Add Readable Stack Traces to Errors
```